sdk_container/src/third_party/coreos-overlay/profiles/coreos/base/package.accept_keywords

```diff
@@ -30,6 +30,16 @@
 
 # Keep versions on both arches in sync.
 =app-emulation/qemu-guest-agent-9.2.0 ~arm64
+
+# Needed to address CVE-2025-22871 and to bring in a patch disabling
+# gold linker on arm64.
+=dev-lang/go-1.24.2-r1 ~amd64 ~arm64
+
+# Needed to address CVE-2024-56406.
+=dev-lang/perl-5.40.2 ~amd64 ~arm64
+
+# Keep versions on both arches in sync.
+=dev-lang/python-3.11.12 ~amd64
```

Comment:
It doesn't really matter, but this is already stable in Gentoo now.

Reply:
Right, but it wasn't on Monday, when these ebuilds were synced. This will be dropped next time we do the sync.